sci-libs/primegen/primegen-0.97.recipe

```diff
@@ -43,10 +43,10 @@ BUILD_REQUIRES="
 	haiku_devel
 	"
 
-TEST_PREREQUIRES="
+TEST_REQUIRES="
 	cmd:md5sum
 	"
-BUILD_PREREQUIRES+="$TEST_PREREQUIRES"
+BUILD_REQUIRES+="$TEST_REQUIRES"
```

Comment:
Oops, you accidentally also remove the "PRE" in BUILD_PREREQUIRES.

Reply:
@pulkomandy said:
> The distinction is relevant only when cross compiling (bootstrap). pre-requires are for running on the host, whereas requires are built for the target. In the case of tests, we can only run them on target, so the distinction is not useful.

I assume then, we would use REQUIRE's?

Reply:
Indeed.